logstash-core/lib/logstash/output_delegator.rb

```diff
@@ -153,10 +168,14 @@ def busy_workers
     if @threadsafe
       0
     else
-      @workers.size - @worker_queue.size
+      @workers.size - (@worker_queue ? @worker_queue.size : 0)
```

Comment:
curious to know a couple of things here:

* why not using `worker_count` like the method you defined in next lines?
* also, will not `@worker_queue.size` return zero when zero? why do you need the if here?

thanks!